<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,2" rowminimumheight="150,300" columnminimumwidth="0">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxInput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Vstup</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,2" columnstretch="1,3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSeed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Násada generátora</string>
         </property>
         <widget class="QRadioButton" name="radioButtonRandomSeed">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>náhodná</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButtonCustomSeed">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>vlastná:</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="labelUsedSeed">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Použitá násada:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelUsedSeedValue">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>100</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBoxParameters">
         <property name="title">
          <string>Parametre simulácie</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Počet replikácií:</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>35</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxSimulationControl">
         <property name="title">
          <string>Ovládanie simulácie</string>
         </property>
         <widget class="QPushButton" name="pushButtonStartSimulation">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Štart</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonStopSimulation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxOutput">
      <property name="title">
       <string>Výstup</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCustomPlot" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
